protocol/src/v4/serializer.rs

```diff
@@ -84,7 +84,9 @@ impl Message {
         cursors[CURSOR_INDEX_MAIN].put_u8(self.hardware_options);
         cursors[CURSOR_INDEX_MAIN].put_u32_be(self.transaction_id);
         cursors[CURSOR_INDEX_MAIN].put_u16_be(self.seconds);
-        cursors[CURSOR_INDEX_MAIN].put_u16_be(if self.is_broadcast { 0x0001 } else { 0x0000 });
+        // https://tools.ietf.org/html/rfc1700
```

Comment:
This link looks more appropriate to me: https://tools.ietf.org/html/rfc2131#section-2

Reply:
RFC2131 doesn't specify bit order, that is there's no "leftmost bit is the most significant bit in network byte order" clause. https://tools.ietf.org/html/rfc1700#page-3 states that explicitly.